python/HeaderService/hutils.py

```diff
@@ -213,13 +213,23 @@ def get_image_size_from_imageReadoutParameters(myData):
     return geom
 
 
-def start_web_server(dirname, port_number=8000, exe='start_www.sh'):
+def start_web_server(dirname, port_number=8000, httpserver="http.server"):
     import subprocess
-    LOGGER.info("Will start web server on dir: {}".format(dirname))
-    subprocess.Popen([exe, dirname, str(port_number)])
-    time.sleep(1)
-    LOGGER.info("Done Starting web server")
-
+    import sys
+    # Get the system's python
+    python_exe = sys.executable
+    # Make sure there isn't another process running
+    cmd = "ps -ax | grep {0} | grep -v grep | awk '{{print $1}}'".format(httpserver)
```

Comment:
I think this code would be significantly more robust if you used the Python `psutil` package.

Reply:
The plan is to move away from the HS starting the www server so we should stop doing this soon. In addtion psutil is not yet installed on the ATHS machine.